tests/integration/source_code/test_jobs.py

```diff
@@ -239,7 +239,7 @@ def test_workflow_linter_lints_job_with_egg_dependency(
         ws.workspace.upload(remote_egg_file, f.read(), format=ImportFormat.AUTO)
     library = compute.Library(egg=remote_egg_file)
 
-    notebook = make_notebook(path=f"{entrypoint}/notebook.ipynb", content=b"import pkgdir")
+    notebook = make_notebook(path=f"{entrypoint}/notebook.ipynb", content=b"from thingy import fn")
```

Comment:
```suggestion
    notebook = make_notebook(path=f"{entrypoint}/notebook.ipynb", content=b"import thingy")
```

To avoid risk of the expected error message not being accurate